src/test/java/net/sf/jabref/gui/keyboard/KeyBindingsDialogViewModelTest.java

```diff
@@ -0,0 +1,175 @@
+package net.sf.jabref.gui.keyboard;
+
+import java.util.prefs.BackingStoreException;
+
+import net.sf.jabref.JabRefPreferences;
+
+import javafx.scene.control.TreeItem;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCombination;
+import javafx.scene.input.KeyEvent;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Test class for the keybindings dialog view model
+ *
+ */
+public class KeyBindingsDialogViewModelTest {
+
+    KeyBindingsDialogViewModel model = null;
+    KeyBindingPreferences keyBindingsPreferences = null;
+    KeyBindingRepository keyBindingRepository = null;
+
+
+    @Before
+    public void setUp() {
+        model = new KeyBindingsDialogViewModel();
+        JabRefPreferences preferences = JabRefPreferences.getInstance();
+        try {
+            preferences.clear();
+        } catch (BackingStoreException e) {
+            e.printStackTrace();
+        }
+        keyBindingsPreferences = new KeyBindingPreferences(preferences);
+        model.setKeyBindingPreferences(keyBindingsPreferences);
+        keyBindingRepository = model.getKeyBindingRepository();
+    }
+
+    @Test
+    public void testNoModifierInvalidKeyBinding() {
```

Comment:
Please change the name so that it reflects what is tested and what is the expected output (here `invalidKeyBindingIsNotSaved`), see wiki on conventions.